data/movie/I Sell the Dead (2008)/i-sell-the-dead/disc01.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
You need to run finalize on this release so the disc json file can be filled out. You can do this by choosing finalize in the ImportBuddy menu or by running:

`.\ImportBuddy.exe finalize "<path to release fodler>"`